homeassistant/components/cybro/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "step": {
+      "user": {
+        "description": "Set up your Cybro PLC to integrate with Home Assistant.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add the Cybro PLC named `{name}` to Home Assistant?",
+        "title": "Discovered Cybro PLC"
+      }
```

Comment:
I don't see zeroconf to be used (not in `manifest.json` and `config_flow.py`. I don't think this is needed?

Reply:
I see you removed zeroconf since your first commit. I think you can add these lines back later when you implement zeroconf.

Reply:
Yes, that's right